### ISSUE TITLE
Missing markdown parsing of image size

### ISSUE THREAD

**Issue by BlackPhlox** [joachimdalen/azdevops-work-item-wiki#18]
`![image.png](/.attachments/image-826f5a2a-8535-45ef-9a6b-19698651b4e8.png =x600)` does not get parsed as an image with a height 600px, nor does it work with `=600x600` & `=600x` :
![image](https://user-images.githubusercontent.com/25123512/174740274-912deaea-a55f-40d7-8615-3aebc9db5a51.png)

And I can't use `<img src="/.attachments/image-826f5a2a-8535-45ef-9a6b-19698651b4e8.png" style="width:600px;"/>` as its not supported in the wiki.

**Comment by joachimdalen** [comment]
Thank you for reporting. I probably won't be able to look at this before the weekend, but I'll see if we can get this fixed then.

**Comment by joachimdalen** [comment]
Just fixed this in `v1.3.2`. 

Now the following formats should parse correctly:

```md
![image.png](/.attachments/image-08e44015-58bd-4ad7-9d49-c74544f42abe.png)
![image.png](/.attachments/image-08e44015-58bd-4ad7-9d49-c74544f42abe.png =x50)
![image.png](/.attachments/image-08e44015-58bd-4ad7-9d49-c74544f42abe.png =100x20)
![image.png](/.attachments/image-08e44015-58bd-4ad7-9d49-c74544f42abe.png =100x)
```
